pyproject.toml

```diff
@@ -0,0 +1,93 @@
+# see documentation, e.g.
+# - https://packaging.python.org/en/latest/specifications/declaring-project-metadata/#declaring-project-metadata
+# - https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html
+# - https://www.python.org/dev/peps/pep-0621/
+
+[build-system]
+requires = ["setuptools>=64.0.0", "setuptools-scm", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+authors = [
+    { name = "Manuel Molano Mazon", email = "manuelmolanomazon@gmail.com" },
+    { name = "Guangyu Robert Yang", email = "gyyang.neuro@gmail.com" } ,
+    { name = "Giulia Crocioni", email = "g.crocioni@esciencecenter.nl" },
+    { name = "Dani Bodor", email = "d.bodor@esciencecenter.nl" },
+    { name = "Jorge Mejias", email = "j.f.mejias@uva.nl"},
```

Comment:
I don't think Jorge will be a contributor on this code (and at least hasn't been for now).

```suggestion
```

Reply:
True, but he is the principal investigator for the annubes component (which basically he has designed), and he will be be the future designated maintainer, at least for that part of the package. Consider also that the authors entries in the zenodo file should align with those listed here.